SEC/Equities/EquityInstruments.rdf

```diff
@@ -406,12 +420,73 @@
 		<skos:definition xml:lang="en">position in an equity instrument</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;EquityRedemptionProvision">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;RedemptionProvision"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasMinimumRedemptionPrice"/>
+				<owl:onClass rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasRedemptionPremium"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasEarliestRedemptionDate"/>
+				<owl:onClass rdf:resource="&fibo-fnd-dt-fd;ExplicitDate"/>
```

Comment:
Is this always absolute? And not relative to the effective date e.g. "after 2 years"?

Reply:
Earliest date is typically a date - it would be helpful if you can explain why you think it would be relative.  That's not what we are seeing in the literature.  And, it's optional, so if there is some sort of rare relative date trigger, someone could add that to the ontology in a specific case using the hasRelativeDate property.

Reply:
I read "Normally, there is a waiting period, often five years, between the stock issue date and the first call date". However I now realize that each Preferred Share has a specific issue date so hence a specific early call date.